src/screen/ReferendaScreen.tsx

```diff
@@ -0,0 +1,68 @@
+import * as React from 'react';
+import {Divider, Layout, Text} from '@ui-kitten/components';
+import SafeView, {noTopEdges} from 'presentational/SafeView';
+import {FlatList, StyleSheet, View} from 'react-native';
+import globalStyles, {standardPadding} from 'src/styles';
+import {useApi} from 'context/ChainApiContext';
+import {useQuery} from 'react-query';
+import {EmptyView} from 'presentational/EmptyView';
+import type {DeriveReferendumExt} from '@polkadot/api-derive/types';
+import {Proposal} from 'presentational/Proposal';
+import {useBestNumber} from 'src/hook/useVotingStatus';
+import {BN_ONE} from '@polkadot/util';
+import {useBlockTime} from 'src/hook/useBlockTime';
+
+export function ReferendaScreen() {
+  const {data, isLoading, refetch} = useReferendums();
+
+  return (
+    <Layout style={globalStyles.flex}>
+      <SafeView edges={noTopEdges}>
+        <View style={styles.container}>
```

Comment:
This `View` can be removed if no styles are needed

Reply:
done ✅